<ui version="4.0" >
 <class>TutorialDialog</class>
 <widget class="QDialog" name="TutorialDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Quick Tutorial</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>12</number>
   </property>
   <item>
    <widget class="QTextBrowser" name="textBrowser" />
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QCheckBox" name="checkStartup" >
       <property name="text" >
        <string>&amp;Show at startup</string>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonBack" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>&lt; &amp;Back</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonNext" >
       <property name="text" >
        <string>&amp;Next ></string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonClose" >
       <property name="text" >
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>textBrowser</tabstop>
  <tabstop>checkStartup</tabstop>
  <tabstop>buttonBack</tabstop>
  <tabstop>buttonNext</tabstop>
  <tabstop>buttonClose</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonClose</sender>
   <signal>clicked()</signal>
   <receiver>TutorialDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>477</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel" >
     <x>430</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
